README.md

```diff
@@ -299,14 +299,27 @@ There is some need for this potentially, as illustrated by the React usecase.
 
 ### Delegating work to blocking workers
 
-This is loosely based off an idea of "transactional JS". This would delegate work to the Host, who
-would run it in the context of a worker and block further execution until the result is ready.
-Objects which require manipulation would need to be isolated (read-only to the worker),
-and the modification could only be applied when the worker returns the result. This approach would
-not invalidate Run to Completion, as the guarentee provided by Run to Completion is that the data
-will not change in memory while the sync function runs unless it is changed by that function itself.
-
-This leaves the issue of blocking the main thread.
+One of the problem of the proposed solution is that all resources have to be loaded ahead, before
+doing a lazy initialization. Frameeworks have defined systems where the resources are not necessarily
+fetched yet, and if it is not fetched then the framework behave as-if we were doing a transaction, by
+unwinding everything and starting over.
```

Comment:
I believe this is only true of react's experimental suspense component -- other frameworks hide the async behavior within the update loop